tests/test_packages/test_skills/test_carpark.py

```diff
@@ -19,151 +19,49 @@
-            standalone_mode=False,
-        )
-        assert result.exit_code == 0
-        result = self.runner.invoke(
-            cli,
-            [*CLI_LOG_OPTION, "create", "--local", self.agent_name_two],
-            standalone_mode=False,
-        )
-        assert result.exit_code == 0
+
+        self.initialize_aea()
+
+        agent_name_one = "my_carpark_aea"
+        agent_name_two = "my_carpark_client_aea"
+        self.create_agents(agent_name_one, agent_name_two)
 
         # Setup agent one
-        agent_one_dir_path = os.path.join(self.t, self.agent_name_one)
+        agent_one_dir_path = os.path.join(self.t, agent_name_one)
         os.chdir(agent_one_dir_path)
-
-        result = self.runner.invoke(
-            cli,
-            [*CLI_LOG_OPTION, "add", "--local", "connection", "fetchai/oef:0.2.0"],
-            standalone_mode=False,
-        )
-        assert result.exit_code == 0
-
-        result = self.runner.invoke(
-            cli,
-            [
-                *CLI_LOG_OPTION,
-                "config",
-                "set",
-                "agent.default_connection",
-                "fetchai/oef:0.2.0",
-            ],
-            standalone_mode=False,
-        )
-        assert result.exit_code == 0
-
-        result = self.runner.invoke(
-            cli,
-            [
-                *CLI_LOG_OPTION,
-                "add",
-                "--local",
-                "skill",
-                "fetchai/carpark_detection:0.1.0",
-            ],
-            standalone_mode=False,
-        )
-        assert result.exit_code == 0
-
-        result = self.runner.invoke(
-            cli, [*CLI_LOG_OPTION, "install"], standalone_mode=False
-        )
-        assert result.exit_code == 0
+        self.add_item("connection", "fetchai/oef:0.1.0")
+        self.add_item("skill", "fetchai/carpark_detection:0.1.0")
+        self.run_install()
 
         # Load the skill yaml file and manually insert the things we need
-        yaml_path = os.path.join(
-            "vendor", "fetchai", "skills", "carpark_detection", "skill.yaml"
-        )
-        file = open(yaml_path, mode="r")
+        setting_path = "vendor.fetchai.skills.carpark_detection.models.strategy.args.db_is_rel_to_cwd"
+        self.set_config(setting_path, False)
 
-        # read all lines at once
-        whole_file = file.read()
-
-        whole_file = whole_file.replace(
-            "db_is_rel_to_cwd: true", "# db_is_rel_to_cwd: true"
-        )
-        whole_file = whole_file.replace(
-            "db_rel_dir: ../temp_files", "# db_rel_dir: ../temp_files"
-        )
+        # Load the agent yaml file and manually insert the things we need (ledger APIs)
 
-        # close the file
-        file.close()
+        # TODO: remove this block and replace with next two commented lines
+        # when "aea config set" will be able to handle dictionaties and non-existing keys
```

Comment:
just write a helper method on the test case which lets you do this by loading and dumping the file